examples/raytrace-parallel/Cargo.toml

```diff
@@ -13,7 +13,7 @@ futures = "0.1"
 js-sys = "0.3.22"
 raytracer = { git = 'https://github.com/alexcrichton/raytracer', branch = 'update-deps' }
 wasm-bindgen = { version = "0.2.45", features = ['serde-serialize'] }
-wasm-bindgen-futures = "0.3.22"
+wasm-bindgen-futures = { path = '../../crates/futures', version = "0.3.22" }
```

Comment:
If this `path` was necessary, mind avoiding this change and instead updating `/Cargo.toml` and the `[patch]` section?